query-engine/connectors/mongodb-query-connector/src/interface/transaction.rs

```diff
@@ -201,17 +201,28 @@ impl<'conn> WriteOperations for MongoDbTransaction<'conn> {
         .await
     }
 
-    async fn execute_raw(&mut self, _inputs: HashMap<String, PrismaValue>) -> connector_interface::Result<usize> {
-        Err(MongoError::Unsupported("Raw queries".to_owned()).into_connector_error())
+    async fn execute_raw(&mut self, inputs: HashMap<String, PrismaValue>) -> connector_interface::Result<usize> {
+        catch(async move { write::execute_raw(&self.connection.database, &mut self.connection.session, inputs).await })
+            .await
     }
 
     async fn query_raw(
         &mut self,
-        _model: Option<&ModelRef>,
-        _inputs: HashMap<String, PrismaValue>,
-        _query_type: Option<String>,
+        model: Option<&ModelRef>,
+        inputs: HashMap<String, PrismaValue>,
+        query_type: Option<String>,
     ) -> connector_interface::Result<serde_json::Value> {
-        Err(MongoError::Unsupported("Raw queries".to_owned()).into_connector_error())
+        catch(async move {
+            write::query_raw(
+                &self.connection.database,
+                &mut self.connection.session,
+                model,
+                inputs,
+                query_type,
+            )
+            .await
+        })
+        .await
```

Comment:
I believe it should also be possible to just forward the call like in the diff below instead. But that also applies to all of the other `ReadOperations` and `WriteOperations`, as far as I can tell — not sure why the other methods don't do it like this. Am I missing something?

```suggestion
        self.connection.query_raw(model, inputs, query_type).await
```

Reply:
Good point, I'm not sure either tbh. Maybe this was not to introduce hidden dependencies between the two interfaces. Let's keep it the way it is for now 🙏